package.json

```diff
@@ -61,6 +75,7 @@
     "prettier": "^2.8.4",
     "pretty-quick": "^3.1.3",
     "protractor": "~7.0.0",
+    "serverless-single-page-app-plugin": "^1.0.4",
```

Comment:
It looks like you're not using the local version of this plugin, but the npm version. Could you please fix that?

Reply:
thank you for the note. fixed